scripts/test

```diff
@@ -6,6 +6,6 @@ if [ -d 'venv' ] ; then
 fi
 
 ${PREFIX}pytest --cov-report term-missing --cov-report=xml:coverage.xml --cov=commitizen tests/
-${PREFIX}black commitizen tests --check
```

Comment:
These shouldn't be removed, these are for "testing" purposes, in order to apply them there's a `lint` script

Reply:
fixed